test/blackbox_tests/outbound_redirect_test.go

```diff
@@ -970,3 +970,185 @@ var _ = Describe("Outbound IPv6 TCP traffic to any address:port except ports exc
 		}(),
 	)
 })
+
+var _ = Describe("Outbound IPv4 TCP traffic from specific interface to other ip than excluded", func() {
+	var err error
+	var ns *netns.NetNS
+	var ns2 *netns.NetNS
+
+	BeforeEach(func() {
+		mainLink, peerLink, linkErr := netns.NewLinkPair()
+		Expect(linkErr).To(BeNil())
+
+		ns1Address, addrErr := netlink.ParseAddr("192.168.0.1/24")
+		Expect(addrErr).To(BeNil())
+		ns, err = netns.NewNetNSBuilder().WithSharedLink(mainLink, ns1Address).Build()
+		Expect(err).To(BeNil())
+
+		ns2Address, addrErr := netlink.ParseAddr("192.168.0.2/24")
+		Expect(addrErr).To(BeNil())
+		ns2, err = netns.NewNetNSBuilder().WithSharedLink(peerLink, ns2Address).Build()
+		Expect(err).To(BeNil())
+	})
+
+	AfterEach(func() {
+		Expect(ns.Cleanup()).To(Succeed())
+		Expect(ns2.Cleanup()).To(Succeed())
+	})
+
+	DescribeTable("should be redirected to outbound port",
+		func(serverPort, randomPort uint16) {
+			// given
+			tproxyConfig := config.Config{
+				Redirect: config.Redirect{
+					Outbound: config.TrafficFlow{
+						Enabled: true,
```

Comment:
Nothing is excluded on the outbound, why?

Reply:
It's more about `vnet` configuration. We create rules that requests that is not a interface IP should be redirected